composer.json

```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^8.0.2",
-        "illuminate/support": "^9.0",
-        "illuminate/http": "^9.0",
-        "monolog/monolog": "^1.12|^2.0",
+        "php": "^8.1.2",
+        "illuminate/support": "^9.21|^10.0",
+        "illuminate/http": "^9.21|^10.0",
```

Comment:
There should not be a minor version declared. what if someone has a Laravel 9 project with version lower than 9.21? I know, it's kinda edge case, but you get the point. This would not work for him